cypress/integration/test_1_runQuery.js

```diff
@@ -0,0 +1,36 @@
+/// <reference types="cypress" />
+import { visitWithToken } from "../integration-helpers/visitWithToken";
+
+const USER_TOKEN_WITH_PERMISSIONS = "user.user2";
+
+describe("Run query", () => {
+  before(() => {
+    visitWithToken(USER_TOKEN_WITH_PERMISSIONS);
+  });
+
+  it("Can execute query and see it in the queries tab", () => {
+    cy.get('[data-test-id="right-pane-container"] >div:visible').as(
```

Comment:
Syntactic question, why does Cypress do naming here instead of just using a variable that you can reference?

Reply:
Ah, because it's good practice to always **select** elements right before doing something with them